src/components/waitlist/WaitlistManager.tsx

```
@@ -0,0 +1,400 @@
+ *
+ * Admin component for managing event waitlists.
+ * Allows promoting users, viewing queue, and configuring settings.
+ */
+
+import { useState, useCallback } from "react";
+import { useEventWaitlist } from "@/hooks/useEventWaitlist";
+import { Button } from "@/components/ui/button";
+import { Input } from "@/components/ui/input";
+import { Switch } from "@/components/ui/switch";
+import { cn } from "@/lib/utils";
+import { formatWaitlistPosition, formatWaitDuration, calculateWaitlistStats } from "@/lib/waitlist-utils";
```

ESLint

[error] Replace `·formatWaitlistPosition,·formatWaitDuration,·calculateWaitlistStats·` with `⏎··formatWaitlistPosition,⏎··formatWaitDuration,⏎··calculateWaitlistStats,⏎`

(prettier/prettier)